src/DataContainer/DcaFilterInterface.php

```diff
@@ -14,6 +14,8 @@
  * DCA filter interface.
  *
  * @author Leo Feyer <https://github.com/leofeyer>
+ *
+ * @deprecated Deprecated since Contao 4.4.1, to be removed in Contao 5.
```

Comment:
Since the interface has only been added in version 4.4.0 and the functionality is broken, we can remove it completely.